pkg/fmap/fmap.go

```diff
@@ -86,6 +86,22 @@ type Metadata struct {
 	Start uint64
 }
 
+func headerValid(h *Header) bool {
```

Comment:
`Header` should be lowercase here.

Reply:
The struct is defined with an upper case. Should it be unexported?

Reply:
Nevermind, you're right! I don't know what I was thinking.